core/web/resolver/query.go

```diff
@@ -409,20 +409,26 @@ func (r *Resolver) ETHKeys(ctx context.Context) (*ETHKeysPayloadResolver, error)
 
 			continue
 		}
-		if err != nil {
-			return nil, fmt.Errorf("error getting EVM Chain: %v", err)
-		}
 
-		ethKeys = append(ethKeys, ETHKey{
-			addr:  k.EIP55Address,
-			state: state,
-			chain: chain,
-		})
+		/*
+			It is not an error to have ETH keys persisted which are not part of a currently
+			configured chain; ergo, this should not cause listing the keys that have currently
+			configured chains to fail
+			TBD: Revisit when we have a comprehensive strategy for what can be 'orphaned keys'
+		*/
+		if err == nil {
+			ethKeys = append(ethKeys, ETHKey{
+				addr:  k.EIP55Address,
+				state: state,
+				chain: chain,
```

Comment:
Can/should we include the keys still, but without `chain` set? b/c we do this already for the special case of `evm.ErrNoChains` above ^

Reply:
It doesn't match up to the operator-ui key management section these keys fall under - titled "EVM Chain Accounts", and it seemed that the 'no chains' condition was a legacy error condition that is no longer possible error - but, importantly, I'm not 100% certain - so I didn't remove it.